zuul.d/tempest_multinode.yaml

```diff
@@ -63,11 +63,47 @@
       - openstack-k8s-operators/repo-setup
       - openstack-k8s-operators/edpm-ansible
 
+- job:
+    name: podified-multinode-edpm-deployment-crc-tempest
+    parent: podified-multinode-edpm-deployment-crc
+    vars:
+      cifmw_extras:
+        - '@scenarios/centos-9/ci.yml'
+        - '@scenarios/centos-9/multinode-ci.yml'
+        - '@scenarios/centos-9/ceph_backends.yml'
+      cifmw_tempest_tempestconf_config:
```

Comment:
I think that this one will override the cifmw_tempest_tempestconf_config definition set in the scenario file